.github/workflows/checks.yaml

```diff
@@ -173,11 +174,9 @@ jobs:
           key:
             ${{ runner.os }}-${{ steps.python.outputs.python-version }}-${{
             needs.prepare-base.outputs.python-key }}
-      - name: Install tox
```

Comment:
This step wasn't necessary because `tox` is installed already via `requirements_test.txt` and it is available once the venv is activated.